.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,20 +35,27 @@
 
 ### Contributor Checklist
 
-- [ ] Contributor has fully tested the PR manually
+
+PR process:
+
 - [ ] PR has the correct target branch and milestone
 - [ ] PR has 'needs review' or 'work-in-progress' label
 - [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
-- [ ] If there are any front-end changes, before/after screenshots are included
 - [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
 
+Testing:
+
+- [ ] Contributor has fully tested the PR manually
+- [ ] If there are any front-end changes, before/after screenshots are included
+- [ ] If this includes an internal dependency change, a link to the diff is provided
```

Comment:
I think this should be moved up to the PR process section.

Reply:
thanks good catch